src/frontend/js/socket/websocket_client.js

```diff
@@ -458,10 +458,11 @@ SeashellWebsocket.prototype.marmosetSubmit = function(project, assn, subdir, def
     deferred);
 };
 
-SeashellWebsocket.prototype.startIO = function(project, pid, deferred) {
+SeashellWebsocket.prototype.startIO = function(project, question, pid, deferred) {
   return this.sendMessage({
     type : "startIO",
     project : project,
+    question : question,
```

Comment:
You're right. I only needed to know the question because I logged to the question folder. If I log to /tmp I don't need to modify this. Thanks Edward!